crepecake/src/main/java/com/google/cloud/tools/crepecake/http/Request.java

```diff
@@ -24,17 +24,17 @@
 public class Request {
```

Comment:
kinda looks like Request should have a builder and be immutable?

Reply:
Changed to builder pattern.